opentracing-api/src/main/java/io/opentracing/propagation/Format.java

```diff
@@ -71,5 +71,16 @@
          * @see Format
          */
         public final static Format<ByteBuffer> BINARY = new Builtin<ByteBuffer>();
+
+        /**
+         * @return Short name for built-in formats as they tend to show up in exception messages.
+         */
+        @Override
+        public String toString() {
```

Comment:
My bad, enum is not possible due to the Format generic.
Made the constructor private instead.